src/explore-education-statistics-admin/src/pages/methodology/MethodologiesPage.tsx

```diff
@@ -121,31 +114,6 @@ const MethodologiesPage = () => {
       </div>
 
       <Tabs id="methodologyTabs">
-        <TabsSection
-          id="live-methodologies"
-          title={`Live methodologies ${
-            model && model.liveMethodologies.length
-              ? `(${model.liveMethodologies.length})`
-              : '(0)'
-          }`}
-        >
-          <Link
-            to="/methodologies/create"
-            className="govuk-button govuk-!-margin-bottom-1"
-          >
-            Create new methodology
-          </Link>
-          {model && model.liveMethodologies.length ? (
-            <MethodologiesTable methodologies={model.liveMethodologies} />
-          ) : (
-            <InsetText>
-              There are currently no live methodologies for you to view.
-            </InsetText>
-          )}
-          <Link to="/methodologies/create" className="govuk-button">
-            Create new methodology
```

Comment:
I think you need to move this button to the other tab.

Reply:
I've moved the button above all the tabs: seems a more sensible place for it.